packages/fxa-react/components/Header/index.tsx

```diff
@@ -7,11 +7,15 @@ import React, { ReactElement } from 'react';
 type HeaderProps = {
   left: ReactElement;
   right: ReactElement;
+  className?: string;
 };
 
 export const Header = (props: HeaderProps) => {
   return (
-    <header role="banner" className="header-page sticky top-0 bg-grey-10">
+    <header
+      role="banner"
+      className={props.className || 'sticky top-0 bg-grey-10'}
```

Comment:
Noice👌 You could also destructure `className` to avoid `props.className` but this works.